debug_toolbar/panels/__init__.py

```diff
@@ -154,6 +154,9 @@ def enable_instrumentation(self):
 
         Unless the toolbar or this panel is disabled, this method will be
         called early in ``DebugToolbarMiddleware``. It should be idempotent.
+
+        Add aenable_instrumentation method to the panel class to support async
+        instrumentation logic.
```

Comment:
maybe an "if needed" ? it's not mandatory for async to have a aenable ?

Reply:
yup, although `async instrumentation logic` itself means that `await` is required in that case `aenable_instrumentation` would be the only choice?